graphbolt/src/unique_and_compact.cc

```diff
@@ -85,5 +85,37 @@ std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> UniqueAndCompact(
 #endif
   return std::tuple(unique_ids, compacted_src_ids, compacted_dst_ids);
 }
+
+std::vector<std::tuple<torch::Tensor, torch::Tensor, torch::Tensor>>
+UniqueAndCompactBatched(
```

Comment:
Consider add regression test cases for this ops.

Reply:
There is already regression test for the unique_and_compact for the non batched case, we can check initial speedup using the existing test. Then, we can add a regression test for the batched variant.